src/Core/Context/CurrentContext.cs

```diff
@@ -382,7 +385,9 @@ public async Task<bool> ViewAssignedCollections(Guid orgId)
         var org = GetOrganization(orgId);
         if (org != null)
         {
-            canCreateNewCollections = !org.LimitCollectionCreationDeletion || org.Permissions.CreateNewCollections;
+            (await _applicationCacheService.GetOrganizationAbilitiesAsync())
+                .TryGetValue(org.Id, out var organizationAbility);
+            canCreateNewCollections = !organizationAbility?.LimitCollectionCreationDeletion ?? org.Permissions.CreateNewCollections;
```

Comment:
If you look a little earlier in this method, this isn't used by FC. I think it was initially, then as we replaced more of the collections controller, FC-specific logic was added to an authorization handler.

Therefore, you should be able to remove the `limitCollectionCreationDeletion` references from here entirely, along with the cache service.

Reply:
Ah - I think you're right. It seems like this method should only be called when the feature flag for MPV is **off**. Will remove with next commit.